dowhy/timeseries/causal_effect_tutorial.ipynb

```diff
@@ -94,54 +90,45 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 4,
+   "execution_count": 3,
    "metadata": {},
    "outputs": [],
    "source": [
-    "from dowhy.timeseries.timeseries import find_lagged_parent_nodes"
+    "from dowhy.timeseries.temporal_shift import find_lagged_parent_nodes,shift_columns,_filter_columns"
```

Comment:
can you provide better node names for the graph? V0, V1, etc. can work. Right now, `1`, `2` as names are confusing because data values are also of similar range.